### ISSUE TITLE
"context-util.h" is missing from tpm2-0-tools-2.1.1.tar.gz release tarball

### ISSUE THREAD

**Issue by liuqun** [intel/tpm2-tools#601]
wget https://github.com/01org/tpm2-tools/releases/download/2.1.1/tpm2-0-tools-2.1.1.tar.gz
tar xzvf tpm2-0-tools-2.1.1.tar.gz
cd tpm2-0-tools-2.1.1
./configure && make
```
$ make
depbase=`echo lib/context-util.o | sed 's|[^/]*$|.deps/&|;s|\.o$||'`;\
gcc -DPACKAGE_NAME=\"tpm2.0-tools\" -DPACKAGE_TARNAME=\"tpm2-0-tools\" -DPACKAGE_VERSION=\"2.1.1\" -DPACKAGE_STRING=\"tpm2.0-tools\ 2.1.1\" -DPACKAGE_BUGREPORT=\"\" -DPACKAGE_URL=\"\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DPACKAGE=\"tpm2-0-tools\" -DVERSION=\"2.1.1\" -DHAVE_TCTI_DEV=1 -DHAVE_TCTI_SOCK=1 -I.    -I./src -I./lib  -Wall -Wextra -Werror -Wformat -Wformat-security -Wstack-protector -fstack-protector-all -D_FORTIFY_SOURCE=2 -U_FORTIFY_SOURCE -fPIC -fPIE -D_GNU_SOURCE -fdata-sections -ffunction-sections   -g -O2 -MT lib/context-util.o -MD -MP -MF $depbase.Tpo -c -o lib/context-util.o lib/context-util.c &&\
mv -f $depbase.Tpo $depbase.Po
lib/context-util.c:44:10: fatal error: context-util.h: No such file or directory
 #include "context-util.h"
          ^~~~~~~~~~~~~~~~
compilation terminated.
Makefile:1548: recipe for target 'lib/context-util.o' failed
make: *** [lib/context-util.o] Error 1
```

```
$ wget https://github.com/01org/tpm2-tools/releases/download/2.1.1/tpm2-0-tools-2.1.1.tar.gz

$ sha256sum tpm2-0-tools-2.1.1.tar.gz
5aeab70e3c4d5a9844878217639d0ca737d67d9e3e8027babe181bfadd8d3277  tpm2-0-tools-2.1.1.tar.gz

$ tar xzvf tpm2-0-tools-2.1.1.tar.gz 
tpm2-0-tools-2.1.1/
tpm2-0-tools-2.1.1/test/
tpm2-0-tools-2.1.1/test/unit/
tpm2-0-tools-2.1.1/test/unit/test_string_bytes.c
tpm2-0-tools-2.1.1/test/unit/tpm2-rc-entry_unit.c
tpm2-0-tools-2.1.1/test/unit/tpm2-rc-decode_unit.c
tpm2-0-tools-2.1.1/test/unit/test_files.c
tpm2-0-tools-2.1.1/Makefile.am
tpm2-0-tools-2.1.1/lib/
tpm2-0-tools-2.1.1/lib/files.c
tpm2-0-tools-2.1.1/lib/password_util.c
tpm2-0-tools-2.1.1/lib/tpm_hash.c
tpm2-0-tools-2.1.1/lib/tpm_hmac.c
tpm2-0-tools-2.1.1/lib/tpm2-header.c
tpm2-0-tools-2.1.1/lib/rc-decode.c
tpm2-0-tools-2.1.1/lib/string-bytes.c
tpm2-0-tools-2.1.1/lib/context-util.c
tpm2-0-tools-2.1.1/lib/tpm_session.c
tpm2-0-tools-2.1.1/lib/options.c
tpm2-0-tools-2.1.1/lib/log.c
tpm2-0-tools-2.1.1/lib/tpm_kdfa.c
tpm2-0-tools-2.1.1/lib/pcr.c
tpm2-0-tools-2.1.1/install-sh
tpm2-0-tools-2.1.1/m4/
tpm2-0-tools-2.1.1/m4/ltsugar.m4
tpm2-0-tools-2.1.1/m4/ltversion.m4
tpm2-0-tools-2.1.1/m4/lt~obsolete.m4
tpm2-0-tools-2.1.1/m4/ltoptions.m4
tpm2-0-tools-2.1.1/m4/libtool.m4
tpm2-0-tools-2.1.1/configure
tpm2-0-tools-2.1.1/test-driver
tpm2-0-tools-2.1.1/tools/
tpm2-0-tools-2.1.1/tools/tpm2_akparse.c
tpm2-0-tools-2.1.1/tools/tpm2_rc_decode.c
tpm2-0-tools-2.1.1/tools/tpm2_nvreadlock.c
tpm2-0-tools-2.1.1/tools/tpm2_nvrelease.c
tpm2-0-tools-2.1.1/tools/tpm2_getmanufec.c
tpm2-0-tools-2.1.1/tools/tpm2_nvdefine.c
tpm2-0-tools-2.1.1/tools/tpm2_listpcrs.c
tpm2-0-tools-2.1.1/tools/tpm2_load.c
tpm2-0-tools-2.1.1/tools/main.c
tpm2-0-tools-2.1.1/tools/tpm2_nvread.c
tpm2-0-tools-2.1.1/tools/tpm2_createprimary.c
tpm2-0-tools-2.1.1/tools/tpm2_send_command.c
tpm2-0-tools-2.1.1/tools/tpm2_getpubak.c
tpm2-0-tools-2.1.1/tools/tpm2_nvwrite.c
tpm2-0-tools-2.1.1/tools/tpm2_unseal.c
tpm2-0-tools-2.1.1/tools/tpm2_evictcontrol.c
tpm2-0-tools-2.1.1/tools/tpm2_takeownership.c
tpm2-0-tools-2.1.1/tools/tpm2_sign.c
tpm2-0-tools-2.1.1/tools/tpm2_listpersistent.c
tpm2-0-tools-2.1.1/tools/tpm2_hmac.c
tpm2-0-tools-2.1.1/tools/tpm2_loadexternal.c
tpm2-0-tools-2.1.1/tools/tpm2_encryptdecrypt.c
tpm2-0-tools-2.1.1/tools/tpm2_getrandom.c
tpm2-0-tools-2.1.1/tools/tpm2_startup.c
tpm2-0-tools-2.1.1/tools/tpm2_dump_capability.c
tpm2-0-tools-2.1.1/tools/tpm2_makecredential.c
tpm2-0-tools-2.1.1/tools/tpm2_rsadecrypt.c
tpm2-0-tools-2.1.1/tools/tpm2_hash.c
tpm2-0-tools-2.1.1/tools/tpm2_readpublic.c
tpm2-0-tools-2.1.1/tools/tpm2_nvlist.c
tpm2-0-tools-2.1.1/tools/tpm2_verifysignature.c
tpm2-0-tools-2.1.1/tools/tpm2_certify.c
tpm2-0-tools-2.1.1/tools/tpm2_getpubek.c
tpm2-0-tools-2.1.1/tools/tpm2_activatecredential.c
tpm2-0-tools-2.1.1/tools/tpm2_create.c
tpm2-0-tools-2.1.1/tools/tpm2_quote.c
tpm2-0-tools-2.1.1/tools/tpm2_rsaencrypt.c
tpm2-0-tools-2.1.1/config.guess
tpm2-0-tools-2.1.1/configure.ac
tpm2-0-tools-2.1.1/AUTHORS
tpm2-0-tools-2.1.1/INSTALL
tpm2-0-tools-2.1.1/missing
tpm2-0-tools-2.1.1/depcomp
tpm2-0-tools-2.1.1/Makefile.in
tpm2-0-tools-2.1.1/compile
tpm2-0-tools-2.1.1/ltmain.sh
tpm2-0-tools-2.1.1/config.sub
tpm2-0-tools-2.1.1/aclocal.m4
```

See also:
https://github.com/01org/tpm2-abrmd/pull/61
https://github.com/01org/tpm2-tss/issues/422

**Comment by liuqun** [comment]
@williamcroberts 

Try using:
```
git archive --prefix="${tag}/" -o "../${tag}.tar.gz" "${tag}"
```
instead of using `make dist`, because "*.h" are not included in "Makefile.am" yet.

**Comment by williamcroberts** [comment]
@liquin header files need to be added to SRC in the Makefile. It looks like the EXTRA_DIST fix for the man pages never made it either. Ill have to cut a 2.1.1 with this fix.

**Comment by williamcroberts** [comment]
Fixed in #615
